### PR TITLE
Added method to remove tracer and the data associated (issue67)

### LINKED ISSUE #67: Add method to remove tracer (and the combinations that involve it)
I would find very convenient to have a method to remove a tracer and its tracer combinations. At this moment the only way I have found is:
1. loop over all the tracer combinations,  check if the selected tracer is present and then remove the combination with `s.remove_selection(tracers=tracers)`. 
2. Then, remove the tracer with `del s.tracers['tracer_to_be_removed']`

If there is a better way of doing this, I have missed it.

### PR DESCRIPTION
**Pull request by carlosggarcia**
Closes #67.

### CODE REVIEW

**Comment by carlosggarcia** [comment]
I was thinking that it might be more useful and closer to `keep_selection` and `remove_selection`, if `remove_tracer` would actually be `remove_tracers` and I added another method `keep_tracers`

File: sacc/sacc.py [hunk 1]
```diff
@@ -641,6 +641,22 @@ def get_tracer_combinations(self, data_type=None):
...
+        Remove the tracer object and its associated data points
+
+        Parameters
+        -----------
+        name: str
+            A string name of a tracer
+
+        """
+        for trs in self.get_tracer_combinations():
+            if (name in trs[0]) or (name in trs[1]):
```
Comment (joezuntz) [inline sacc/sacc.py:655]: Same comment as other PR!

**Review by joezuntz — CHANGES_REQUESTED** [review]
Same comment as last PR - if you can tweak slightly to avoid assuming that there are two tracers then perfect.

File: sacc/sacc.py (hunk 1 above)
Comment (carlosggarcia) [reply sacc/sacc.py]: Done!

**Comment by carlosggarcia** [comment]
@joezuntz back to you. Thanks for reviewing! In the end, I've added `keep_tracers` and `remove_tracers`. It seems more versatile.

**Review by joezuntz — APPROVED** [review]

**Comment by joezuntz** [comment]
Thanks!

**Pull request merged**